pkg/reconciler/inmemorychannel/dispatcher/inmemorychannel.go

```diff
@@ -19,6 +19,11 @@ package dispatcher
 import (
 	"context"
 
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	messagingv1 "knative.dev/eventing/pkg/client/clientset/versioned/typed/messaging/v1"
```

Comment:
This one belongs with other `knative.dev` imports.

Reply:
new editor, whodis? :) Been playing with goland and obvs I haven't configured it quite as nicely as my emacs :)